README.TXT

```diff
@@ -1,3 +1,32 @@
+Ultimate Tapan Kaikki
+=====================
+
+This is the legendary 1990's game Ultimate Tapan Kaikki, ported to
+SDL2. It runs at least on macOS, Linux, FreeBSD and Windows.
+
+Suomipelit GitHub: https://github.com/suomipelit
+Suomipelit Slack: https://tinyurl.com/suomipelit-slack
+
+
+Build requirements
+------------------
+
+- CMake
+- C++ compiler: At least gcc, clang and Visual Studio are supported
+- Libraries: SDL2, SDL2_mixer
+
+
+Building and running
+--------------------
+
+cmake .

```

Comment:
For Windows one needs to provide SDL libraries explicitly

`cmake -DSDL2_PATH="C:\\<path>\\SDL2-2.0.9" -DSDL2_MIXER_PATH="C:\\<path>\\SDL2_mixer-2.0.4"`

and download and extract runtime libraries to running directory. But since SDL installation is outside of this README's scope, I think mentioning the cmake command only is sufficient.

Reply:
Added.